examples/02_decoding/plot_haxby_searchlight_surface.py

```diff
@@ -9,66 +9,74 @@
 
 """
 
+
+
+
 #########################################################################
 # Load Haxby dataset
 # -------------------
 import pandas as pd
+
 from nilearn import datasets
 
 # We fetch 2nd subject from haxby datasets (which is default)
 haxby_dataset = datasets.fetch_haxby()
 
 fmri_filename = haxby_dataset.func[0]
 labels = pd.read_csv(haxby_dataset.session_target[0], sep=" ")
-y = labels['labels']
-session = labels['chunks']
+y = labels["labels"]
+session = labels["chunks"]
 
 #########################################################################
 # Restrict to faces and houses
 # ------------------------------
 from nilearn.image import index_img
-condition_mask = y.isin(['face', 'house'])
+
+condition_mask = y.isin(["face", "house"])
 
 fmri_img = index_img(fmri_filename, condition_mask)
 y, session = y[condition_mask], session[condition_mask]
 
+from sklearn import neighbors
+
 #########################################################################
 # Surface bold response
 # ----------------------
 from nilearn import datasets, surface
-from sklearn import neighbors
 
 # Fetch a coarse surface of the left hemisphere only for speed
-fsaverage = datasets.fetch_surf_fsaverage(mesh='fsaverage5')
-hemi = 'left'
+fsaverage = datasets.fetch_surf_fsaverage(mesh="fsaverage5")
+hemi = "left"
 
 # Average voxels 5 mm close to the 3d pial surface
-radius = 5.
-pial_mesh = fsaverage['pial_' + hemi]
+radius = 5.0
+pial_mesh = fsaverage[f"pial_{hemi}"]
 X = surface.vol_to_surf(fmri_img, pial_mesh, radius=radius).T
 
-# To define the :term:`BOLD` responses to be included within each searchlight "sphere"
+# To define the :term:`BOLD` responses
+# to be included within each searchlight "sphere"
 # we define an adjacency matrix based on the inflated surface vertices such
 # that nearby surfaces are concatenated within the same searchlight.
 
-infl_mesh = fsaverage['infl_' + hemi]
+infl_mesh = fsaverage[f"infl_{hemi}"]
 coords, _ = surface.load_surf_mesh(infl_mesh)
-radius = 3.
+radius = 3.0
 nn = neighbors.NearestNeighbors(radius=radius)
 adjacency = nn.fit(coords).radius_neighbors_graph(coords).tolil()
 
+from sklearn.linear_model import RidgeClassifier
```

Comment:
I don't understand why this import statement was moved here... It should be below the comment (which will be parsed as the section title), right ?

Reply:
I have tried to move the import in sensible place and re ran isort to make sure it was happy with it.